dist/.htaccess

```diff
@@ -9,14 +9,12 @@ RewriteCond %{SCRIPT_FILENAME} !-f
 rewriterule ^projects/modbod3d(.*)$ http://codame.com/projects/modbod$1 [r=301,nc]
 rewriterule ^artex(.*)$ http://codame.com/events/artificial-experiences$1 [r=301,nc]
 
-# remove trailing slash
-RewriteRule (.*)/$ /$1 [L,R=301]
-
```

Comment:
this will be resolved once https://github.com/CODAME/codame-website/pull/19 is merged